ee/app/livechat-enterprise/server/lib/VisitorInactivityMonitor.js

```diff
@@ -77,11 +78,28 @@ export class VisitorInactivityMonitor {
 		});
 	}
 
+	placeRoomOnHold(room) {
+		LivechatEnterprise.placeRoomOnHold(room);
```

Comment:
This method should be boolean so, you'd only call the next method if this returns success(true).

Reply:
Okay makes sense. I've changed it to 
`LivechatEnterprise.placeRoomOnHold(room) && LivechatRooms.unsetPredictedVisitorAbandonmentByRoomId(room._id);
`